src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -183,6 +186,11 @@ public CassandraStorage(ReaperApplicationConfiguration config, Environment envir
 
     // https://docs.datastax.com/en/developer/java-driver/3.5/manual/metrics/#metrics-4-compatibility
     cassandraFactory.setJmxEnabled(false);
+    if (!CassandraStorage.firstAttempt.get()) {
```

Comment:
nit-picking: this should be 
```
if (!firstAttempt.compareAndSet(true, false)) {
      // If there's been a past connection attempt, metrics are already registered
      cassandraFactory.setMetricsEnabled(false);
}

cassandra = cassandraFactory.build(environment);
```
(and to nit-pick on the nit-pick… i'd be calling the static variable `firstInstantiation` as it's not an attempt: `there is no try. do or do not.`)

Reply:
fair enough, I've applied the suggested changes.